spec/rubocop/cop/rails/reversible_migration_spec.rb

```diff
@@ -169,6 +169,11 @@ def change
       remove_foreign_key :accounts, :branches
     RUBY
 
+    it_behaves_like 'accepts',
+                    'remove_foreign_key(with to_table option)', <<-RUBY
```

Comment:
Can you use squiggly heredoc?

```suggestion
                    'remove_foreign_key(with to_table option)', <<~RUBY
```

Reply:
I found with squiggly heredoc another style rule complained about indentation (see other >80 char workarounds earlier in the file), but have shortened spec label and it fits now.